src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -194,18 +219,34 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
     /**
      * Method called when deleting a region from the editor
      * @param {string} id the id of the deleted region
-     * @param {boolean} multiselection boolean whether multiselect is active
+     * @param {boolean} multiselect boolean whether region was selected with multiselection
      * @returns {void}
      */
     private onRegionSelected = (id: string, multiselect: boolean) => {
-        const region = this.state.currentAsset.regions.find((region) => region.id === id);
+        if (!id) {
```

Comment:
How can `onRegionSelected` get called without an id?

Reply:
This is a CT bug I've mentioned to Konstantin. When a region is deselected, id is undefined. Not sure why, this is a blocker for multiselect to work. Hoping he can get a fix in for this